File: recordAnalyseSystemSparkCode/src/main/java/j/HbaseConf.java
package j;

import org.apache.hadoop.conf.Configuration;

/**
 * Author: cwz
 * Time: 2017/9/20
 * Description: 设置hbase连接配置
 */
public class HbaseConf {

    // 配置hbase的主节点ip

    private final static String master = "192.168.1.10";
    // 配置hbase的zookeeper节点ip
    private final static String zookeeper = "192.168.1.10,192.168.1.11,192.168.1.12";
    // 配置hbase的zookeeper连接端口，默认值为2181
    private final static String port = "2181";

    public static Configuration getConf() {
        Configuration hbaseConf = new Configuration();
        hbaseConf.set("hbase.master", master);
        hbaseConf.set("hbase.zookeeper.quorum", zookeeper);
        hbaseConf.set("hbase.zookeeper.property.clientPort", port);
        return hbaseConf;
    }

}

File: recordAnalyseSystemSparkCode/src/main/scala/s/HbaseConf.scala
package s

import org.apache.hadoop.conf.Configuration

/**
  * Author: cwz
  * Time: 2017/9/19
  * Description: 设置hbase连接配置
  */
object HbaseConf {
  // 配置hbase的主节点ip
  val master: String = "192.168.0.41"
  // 配置hbase的zookeeper节点ip
  val zookeeper: String = "192.168.0.41,192.168.0.42,192.168.0.43"
  // 配置hbase的zookeeper连接端口，默认值为2181
  val port = "2181"

  def getConf(): Configuration = {
    val hbaseConf: Configuration = new Configuration
    hbaseConf.set("hbase.master", master)
    hbaseConf.set("hbase.zookeeper.quorum", zookeeper)
    hbaseConf.set("hbase.zookeeper.property.clientPort", port)
    hbaseConf
  }
}
